src/version.c

```diff
@@ -176,10 +176,9 @@ static int get_version_from_url(char *url)
 	if (globals.sigcheck) {
 		sig_verified = verify_signature(url, &tmp_version);
 	} else {
-		error("FAILED TO VERIFY SIGNATURE OF %s. Operation proceeding due to \n"
-		      " --nosigcheck, but system security may be compromised\n",
-		      url);
-		journal_log_error("swupd security notice:  --nosigcheck used to bypass version signature");
+		warn("The --nosigcheck flag was used, THE SIGNATURE OF %s WILL NOT BE VERIFIED\n", url);
+		info("Be aware that this compromises the system security\n\n");
```

Comment:
I think we need this: info("Be aware that this compromises the system security\n"); You have two line feeds

Reply:
those two lines "\n" are on purpose, I usually add two line breaks when I print important errors to highlight them. I think it looks nicer in the output when the error message is separated from the normal workflow.